db/versions/0017-migration.sql

```diff
@@ -0,0 +1,25 @@
+begin
+  -- lock this table before reading from it, to prevent loss of concurrent
+  -- updates when the table is dropped.  Note that this may lead to concurrent
+  -- updates failing; the important thing is that they not succeed without
+  -- taking effect.  Failed updates will be retried.
+  lock table denylisted_notification_entities;
+
+  create table denylisted_notification
```

Comment:
Most (all?) of our non-entities tables have plural names -- `github_builds`, `tasks`, `worker_pools`.  For consistency, let's do the same with this table.

Reply:
ok